android/src/main/java/com/twiliorn/library/CustomTwilioVideoView.java

```diff
@@ -461,7 +461,9 @@ public void connectToRoomWrapper(
         this.enableH264Codec = enableH264Codec;
 
         // Share your microphone
+        if(enableAudio){
```

Comment:
If I'm understanding this correctly, this means that if the participant joins the call without audio enabled, and subsequently enables it, that things will fail.  

I'm wondering if it would make more sense to have a createAudioTrack flag.  Either that or ensure that when the enabling is toggled that it creates a local audio track if it doesn't already exist.

Reply:
Not it's not the same as you understand . 

suppose user not give microphone permission than this extra code was running so we just run this particular code . 

and it's working fine 
1. when user given permission in between and enabled the audio 
2. And when user already given permission and enable mic turn on and off in between call @slycoder

Reply:
Perhaps I'm confused then;  this is the only place in the code where `LocalAudioTrack.create` is called.  So if the local participant enabled the mic during the call, where would the track get created?

Reply:
May be we can connect related to this if you have bandwidth  @slycoder

Reply:
I'm afraid I don't have time; I'd prefer to resolve this asynchronously if possible.

Reply:
This will not effect we are using same things since many day in our project and it's not creating any issue working fine 
not please test at your end and merge it if are comfortable with this Thanks  @slycoder

Reply:
I just want to make sure that this change doesn't break the use case that some users of the library have where they have permissions but nonetheless want to join the call with the audio initially disabled, but the user can at any time during the call enable the audio.

Reply:
Yes it will not break . @slycoder

Reply:
LocalAudioTrack.create(getContext(), enableAudio) 

This will break the code when permission is not granted @slycoder 

That's why we are creating when user enable the audio